dea_check.py

```diff
@@ -57,10 +42,9 @@
                      'TSCPOS': [(1, 2.5),
                                 (99, 2.5)]
                      }
+HIST_LIMIT = [20.]
```

Comment:
The limits for the individual MSIDs (e.g., 1DEAMZT) should probably live in the tool, since they can be different, and the limits on the histogram are different between DEA, DPA, and PSMC. 

I suppose they will always be the same for TSCPOS, so in theory that could be moved into model_check.